bin/install-ubuntu

```diff
@@ -13,4 +13,6 @@ apt-get update
 DEBIAN_FRONTEND=noninteractive apt-get install --yes python-pip python-dev libxml2-dev libxslt-dev libffi-dev
 apt-get remove --yes ansible || echo "ok"
 
-pip install -r -U $BINDIR/../requirements.txt
+pip install -r requirements.txt
```

Comment:
Sure - I could revert that and use the above command. Was going for consistency between files. Keeping the BINDIR  in place likely makes sense.